pkg/webhook/resources/virtualmachinerestore/validator.go

```diff
@@ -120,6 +124,15 @@ func (v *restoreValidator) Create(request *types.Request, newObj runtime.Object)
 	return v.handleExistVM(newVM, vm)
 }
 
+func (v *restoreValidator) Update(request *types.Request, oldObj, newObj runtime.Object) error {
+	oldRestore := oldObj.(*v1beta1.VirtualMachineRestore)
+	newRestore := newObj.(*v1beta1.VirtualMachineRestore)
+	if !reflect.DeepEqual(oldRestore.Spec, newRestore.Spec) {
+		return werror.NewInvalidError("VirtualMachineRestore spec is immutable", fieldTargetName)
```

Comment:
seems like we need to use `fieldSpec` here instead of `fieldTargetName`

Reply:
Yes, udpated it. Thank you.